.scivision-config_imagenet.yaml

```diff
@@ -0,0 +1,10 @@
+name: test
+url: https://github.com/acocac/scivision-test-plugin.git
+import: scivision_test_plugin
+model: ImageNetModel
+args: None
```

Comment:
We could add the model name in the config file:

```suggestion
args:
    model_name: resnet18
```

Reply:
Cool! It's now clear for me how to declare model arguments in the YAML file.